WalletWasabi.Fluent/Styles/Button.axaml

```diff
@@ -74,4 +75,22 @@
     <Setter Property="Background" Value="{DynamicResource InvisibleButtonPressedBackgroundColor}" />
   </Style>
 
+  <Style Selector="Button.quickAction">
```

Comment:
Nit: I am not sure about `quickAction` is the proper name, it is more like a `plainHyperLink` or something similar.